docs/content/docs/research/01_spec_language_design.md

```
@@ -2531,37 +2529,37 @@ service OrderService {
 Primitive types have no subtypes. Each primitive can be refined with a `where` clause to create a
 named constrained type.
 
-### 5.2 Compound Types
+### 5.2 Compound types
 
-**Set[T]** -- An unordered collection of unique values of type T.
+`Set[T]` is an unordered collection of unique values of type T.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)